app-portage/gpyutils/files/implementations.txt

```diff
@@ -20,8 +20,8 @@ python3_7		dead	3.7
 python3_8		dead	3.8
 python3_9		dead	3.9
 python3_10		supported	3.10
-python3_11		current	3.11
-python3_12		supported	3.12
+python3_11		supported	3.11
```

Comment:
I'm wondering if we should mark 3.10+3.11 as "old". It feels kinda weird to have CPy 3.10 as "old" but PyPy 3.10 as "supported", though.

Reply:
…or maybe 3.10+3.11 as "old", 3.12 "current", 3.13 + pypy3 "experimental".

Reply:
> …or maybe 3.10+3.11 as "old", 3.12 "current", 3.13 + pypy3 "experimental".

I prefer that actually.

Reply:
Yeah, switched to that instead.